streams.yml

```diff
@@ -5,7 +5,20 @@ ose-cli:
 rhel-9-golang:
   aliases:
     - rhel-9-golang-{GO_LATEST}
-  image: openshift/golang-builder:v1.24.4-202507171054.g2f6f49f.el9
+  image: registry.redhat.io/ubi9/go-toolset:1.23
+
+rhel-9-nodejs-18:
+  image: registry.redhat.io/ubi9/nodejs-18:latest
+
+rhel-9-nodejs-20:
+  image: registry.redhat.io/ubi9/nodejs-20:latest
+
+rhel-9-ansible-operator:
+  image: registry.redhat.io/openshift4/ose-ansible-rhel9-operator:v4.17
+
+# Don't know if I can do this with brew registry but will try
+mta-analyzer-lsp:
+  image: brew.registry.redhat.io/rh-osbs/mta-mta-analyzer-lsp-rhel9:8.0.0
```

Comment:
_I think..._ Fine for now to get us going.
We'll come back around to it once we have builds on Konflux via ART.